LunchTime/LunchTime.csproj

```diff
@@ -1,22 +1,81 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
   </PropertyGroup>
 
 
   <ItemGroup>
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="5.0.0" />
```

Comment:
Is there a reason to use Autofac? I would prefer standard .Net Core DI container and for complex tasks using Scrutor...

Reply:
discussed: Scrutor will be more suitable for registration.